### PR TITLE
Convert warning to info

### PR DESCRIPTION
**Pull request by ramonakira**

### CODE REVIEW

File: leukeleu_django_gdpr/checks.py
Comment (jaap3) [inline leukeleu_django_gdpr/checks.py:13]:
```suggestion
    id="gdpr.I001",
```

**Review by jaap3 — CHANGES_REQUESTED** [review]
There's also a reference to W001 in the README (https://github.com/leukeleu/leukeleu-django-gdpr/blob/master/README.md#checks), that needs to be changed as well.

**Review by jaap3 — APPROVED** [review]
lgtm, too bad about the whitespace changes in the README. Please wait for approval of @wadevries before merging

**Review by wadevries — APPROVED** [review]

**Pull request merged**
